projects/scripts/adi_board.tcl

```diff
@@ -7,7 +7,10 @@ package require math
 
 ## Global variables for interconnect interface indexing
 #
+set sys_hpm0fpd_interconnect_index 0
```

Comment:
Do we need sys_hpm0fpd_interconnect_index and sys_hpm1fpd_interconnect_index?

Reply:
not really otherwise you can't keep track of the index independently if you use 2 interfaces at the same time.

Reply:
I could drop all of the variables and "walk through" all ports and add to the first available one though

Reply:
I would leave it up to the caller script to provide the desired index, instead of automating.